hobbit-gui/gui-serverbackend/src/main/java/de/usu/research/hobbit/gui/rest/LogsResources.java

```diff
@@ -132,6 +145,41 @@ public Response query(@QueryParam("id") String id, @Context SecurityContext sc)
         return Response.ok(logs).build();
     }
 
+    private Model getExperimentModel(String experimentId) {
+        String experimentUri = Constants.EXPERIMENT_URI_NS + experimentId;
+        String query = SparqlQueries.getExperimentGraphQuery(experimentUri, null);
+        Model model = StorageServiceClientSingleton.getInstance().sendConstructQuery(query);
+        return model;
+    }
+
+    private Response checkAccessAllowed(Model experimentModel, String experimentId, SecurityContext sc) {
+        String experimentUri = Constants.EXPERIMENT_URI_NS + experimentId;
+        // get the date info from model and restrict query by time
+        if (experimentModel != null && experimentModel.size() > 0) {
+            ExperimentBean experiment = RdfModelHelper.createExperimentBean(experimentModel, experimentModel.getResource(experimentUri));
+            // Check whether the user is the owner of the system
+            String systemURI = experiment.getSystem().getId();
+            Set<String> userOwnedSystemIds = InternalResources.getUserSystemIds(sc);
+            if(!userOwnedSystemIds.contains(systemURI)) {
+                // The user is not allowed to see the systems log
+                return Response.status(Status.FORBIDDEN).build();
+            }
+        } else {
+            return Response.status(Status.NO_CONTENT).build();
+        }
+        return null;
+    }
+
+    private String getExperimentDate(String experimentId) {
+        Model experimentModel = getExperimentModel(experimentId);
+        String experimentUri = Constants.EXPERIMENT_URI_NS + experimentId;
+        Resource experimentResource = experimentModel.getResource(experimentUri);
+        Statement startTime = experimentModel.getProperty(experimentResource, HOBBIT.startTime);
+        String utcString = startTime.getLiteral().getValue().toString();
+        String[] timeParts = utcString.split("T");
+        return timeParts[0];
```

Comment:
It might be better to use the already tested methods we have developed before:
```java
Calendar cal = getTolerantDateTimeValue(experimentModel, experimentResource, HOBBIT.startTime);
if(call != null) {
  return Calendar.getDayOfMonth(); // or whatever you need
} else {
  return ""; // What do we return in case of an error?
}
```